www/js/ui.js

```diff
@@ -1,11 +1,12 @@
-/* window focus/blur */
-$(window).focus(function() {
-    FOCUSED = true;
-    clearInterval(TITLE_BLINK);
-    TITLE_BLINK = false;
-    document.title = PAGETITLE;
-}).blur(function() {
-    FOCUSED = false;
+document.addEventListener("visibilitychange", function() {
```

Comment:
What was the motivation for this change?  I can think of at least a couple issues that need clarification:

  * The [spec](https://www.w3.org/TR/page-visibility/) for `visibilityState` is relatively recent (2013).  There are a number of users who will have browsers too old to support this (people using old Androids / built in browsers on game consoles / etc.).  It would be better to at least fall back to the old `focus` event for these browsers.
  * It seems like according to the spec, this behaves differently than the code that was there before.
  > For example, in the following cases the visibilityState attribute would return the DOMString hidden:

   > The User Agent is minimized.
   > The User Agent is not minimized, but the page is on a background tab.
   > The Operating System lock screen is shown.

  How does this behave when the window is fully visible, but not selected?  For example, if I have a browser on one monitor, and a game on the other, and the game is focused?